pkg/build/gobuild_test.go

```diff
@@ -640,7 +640,13 @@ func TestGoBuildNoKoData(t *testing.T) {
 	})
 }
 
-func validateImage(t *testing.T, img oci.SignedImage, baseLayers int64, creationTime v1.Time, checkAnnotations bool, expectSBOM bool) {
+type validationOptions struct {
+	checkAnnotations   bool
+	expectSBOM         bool
+	entryPointBasePath string
+}
+
+func validateImage(t *testing.T, img oci.SignedImage, baseLayers int64, creationTime v1.Time, opts validationOptions) {
```

Comment:
note that `baseLayers` being 3 in all test uses triggers the linter - as it's unrelated to the change, I didn't want to change the function to appease the linter. lmk if I should